setup.py

```diff
@@ -18,8 +22,7 @@
     'numpy>=1.10.0',
     'scipy>=0.18.0',
     'multipledispatch>=0.4.9',
-    'tabulate',
-    'gast==0.2.2',
+    'tabulate'
 ]
 
 if not is_py37:
```

Comment:
NB- this will break with python 3.8. it should instead be a check on `if is_less_than_py37`... (not your change I know, but wanted to flag it up while we're discussing the setup script!)

Reply:
Hi @st-- ,

Which bit will break exactly? Adding tabulate?

Reply:
no, the line I commented on: `if not is_py37`.

Reply:
don't worry about it here, I've made a separate issue (#1354).